include/exadg/solvers_and_preconditioners/multigrid/multigrid_input_parameters.cpp

```diff
@@ -143,6 +143,27 @@ enum_to_string(MultigridSmoother const enum_type)
   return string_type;
 }
 
+std::string
+enum_to_string(AMGType const enum_type)
+{
+  std::string string_type;
+
+  switch(enum_type)
+  {
+    case AMGType::Trilinos:
+      string_type = "Trilinos";
+      break;
+    case AMGType::Boomer:
+      string_type = "Boomer";
```

Comment:
ML and BoomerAMG

Reply:
Done in commit [d0a2ff9](https://github.com/exadg/exadg/pull/58/commits/d0a2ff93d87cd124a4426a80462886196481f72a).